src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -17,6 +34,11 @@
     abstract void perform();
 
     abstract String getLinkText();
+
+    void showError(@NotNull String title, @NotNull String message) {
+      //TODO(pq): implement
```

Comment:
Right.  I looked at FlutterMessages and it's not quite right.  Alternatively, we do have the error status line in the wizard already so maybe just writing there?